fastly/objectstorage/accesskeys/api_list_access_keys.go

```diff
@@ -0,0 +1,25 @@
+package accesskeys
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+)
+
+// ListAccessKeys retrieves all access keys within object storage that this Fastly api key has access to.
```

Comment:
Maybe it is better to remove the info about the api key to keep it short and simple.